examples/standard/main.tf

```
@@ -35,4 +35,7 @@ module "azureBastion" {
   file_copy_enabled           = true
   tunneling_enabled           = false
   shareable_link_enabled      = false
+  tags                        = {
+    "env" = "PRD"
```

```suggestion
    "tagname" = "tagvalue"
```

Let's not use real env in examples :D

---

Applied recommendation.